modules/vsphere-problem-detector-running.adoc

```diff
@@ -18,41 +18,39 @@ The {operator-name} automatically runs the checks every hour. However, when the
 
 .Procedure
 
-. Scale the Operator to `0`:
+* Scale the Operator to `0`:
 +
 [source,terminal]
 ----
 $ oc scale deployment/vsphere-problem-detector-operator --replicas=0 \
     -n openshift-cluster-storage-operator
 ----
-+
-If the deployment does not scale to zero immediately, you can run the following command to wait for the pods to exit:
-+
-[source,terminal]
-----
-$ oc wait pods -l name=vsphere-problem-detector-operator \
-    --for=delete --timeout=5m -n openshift-cluster-storage-operator
-----
 
-. Scale the Operator back to `1`:
-+
-[source,terminal]
-----
-$ oc scale deployment/vsphere-problem-detector-operator --replicas=1 \
-    -n openshift-cluster-storage-operator
-----
+.Verification
 
-. Delete the old leader lock to speed up the new leader election for the Cluster Storage Operator:
+. Verify that the pods have restarted by running the following command:
 +
 [source,terminal]
 ----
-$ oc delete -n openshift-cluster-storage-operator \
-    cm vsphere-problem-detector-lock
+$ oc -n openshift-cluster-storage-operator get pod -l name=vsphere-problem-detector-operator
 ----
-
-.Verification
-
-* View the events or logs that are generated by the {operator-name}. Confirm that the events or logs have recent timestamps.
++
+.Example output
+|===
+|NAME
+|READY
+|STATUS
+|RESTARTS
+|AGE
+
+|vsphere-problem-detector-operator-77486bd645-9ntpb
+|1/1
+|Running
+|0
+|11s
+|===
+
```

Comment:
@snarayan-redhat Hi, the output of the command doesn't good, it should be as below, but the docs preview gives a table, could you check please?
```
NAME                                                 READY   STATUS    RESTARTS   AGE
vsphere-problem-detector-operator-77486bd645-9ntpb   1/1     Running   0          11s
```

Reply:
Thanks for the catch. It seems to look fine now. Can you give another look?